stackSentinel_rsmas.py

```diff
@@ -1,938 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
stackSentinel_rsmas.py is no longer needed. we will use the original script of stackSentinel.py